pages/outfit-wars/index.vue

```diff
@@ -464,12 +466,23 @@ export default Vue.extend({
       }
 
       console.log('OutfitwarsRankings.pull')
+      const currentRoundRequests = [];
+      for(const world of this.worlds) {
+        currentRoundRequests.push(new ApiRequest()
+          .get<number>(ps2AlertsApiEndpoints.outfitwarsCurrentRoundByWorld.replace('{world}', world.toString())))
+      }
+
+      await Promise.all(currentRoundRequests).then((result) => {
+        this.worldRounds = result;
+      })
+
+      console.log(this.worldRounds);
 
       await new ApiRequest()
         .get<OutfitwarsRankingInterface[]>(Endpoints.OW_RANKINGS_ALL)
         .then(async (result) => {
           console.log('result', result)
-          await this.parse(result)
+          this.parse(result)
```

Comment:
You sure removing await here is correct? As the containing method at 483 is async